Xamarin.Essentials/Contacts/Contact.ios.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Contacts;
+using ContactsUI;
+using Foundation;
+using UIKit;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Contact
+    {
+        internal static Action<PhoneContact> CallBack { get; set; }
+
+        internal static UIViewController UIView => Platform.GetCurrentViewController();
```

Comment:
I would say not needed and just call it below in each method, could also check against null.

Reply:
done.